emails/templates/emails/first_time_user.html

```diff
@@ -294,7 +294,7 @@ <h1 style="margin-bottom: 30px; line-height: 140%; text-align: center; word-wrap
                           </p>
 
                           <p style="line-height: 140%; text-align: left; word-wrap: break-word;">
-                            {% ftlmsg 'first-time-user-email-how-item-1-subhead-html' url='https://addons.mozilla.org/firefox/addon/private-relay/?utm_campaign=relay-onboarding&utm_source=relay-onboarding&utm_medium=email&utm_content=addon-cta' %}
+                            {% ftlmsg 'first-time-user-email-how-item-1-subhead-html' url='https://addons.mozilla.org/firefox/addon/private-relay/?utm_campaign=relay-onboarding&utm_source=relay-onboarding&utm_medium=email&utm_content=addon-cta' attrs='' %}
```

Comment:
question (non-blocking): do we have to, or is it best practice, to include arguments even when they are meant to be blank?